adding titles to TOC and landing pages for everything other than Euclid (#220)
Caltech-IPAC/irsa-tutorials: tutorials/techniques.md

# Techniques & Tools Tutorial Notebooks

This section gathers tutorials that go beyond mission specific data access and analysis to explore broader methods, workflows, and utilities applicable across missions and datasets. 
This collection includes guidance on cloud-based data access, use of powerful big data tools such as lsdb, interactive visualization tools, and computational techniques like parallel image convolution.

- [Cloud Access](cloud_access/cloud-access-intro.md) — Introduces how to access and work with data served by IRSA from AWS S3 cloud storage.

> - [LSDB HATS](parquet-catalog-demos/irsa-hats-with-lsdb.md) - Uses LSDB and HATs framework for advanced cross-matching and query strategies

Reviewer: ```suggestion
- [HATS with LSDB](parquet-catalog-demos/irsa-hats-with-lsdb.md) - Uses LSDB for user-friendly cross matching and querying of HATS Collections.
```

Seems to me that "user friendly" is a key selling point of LSDB, especially for cross matching.

> - [SED visualization](firefly/SEDs_in_Firefly.md) - Demonstrates how to explore and interactively visualize SEDs using Firefly on multi-wavelength datasets.

Reviewer: ```suggestion
- [SED Visualization](firefly/SEDs_in_Firefly.md) - Demonstrates how to explore and interactively visualize SEDs using Firefly on multi-wavelength datasets.
```
